docs/release-notes/release-notes-0.8.1.md

```diff
@@ -0,0 +1,163 @@
+Joinmarket-clientserver 0.8.1:
+=================
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/releases/tag/v0.8.1>
+
+This release contains some important new features and some fairly important bugfixes/improvements to performance. Upgrade is recommended as soon as possible.
+
+A note that backing up and then recreating a default `joinmarket.cfg` file is *always* a good idea for a new release; make sure to do it for this release, so that all default values and comments are populated. A couple of new config settings now exist, which you should take note of.
+
+Please report bugs using the issue tracker at github:
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/issues>
+
+Upgrading 
+=========
+
+To upgrade:
+
+(If you are upgrading from a version pre-0.6.2 then please see the section "Move user data to home directory" in [the 0.6.2 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md), and follow the instructions there (except the `commitmentlist` file - that can actually be left alone, the previous release notes were wrong on this point).)
+
+(If you are upgrading from a version pre-0.7.0 please read the "Upgrading" section in [the 0.7.0 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.7.0.md).)
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### Yield Generator config moved to `joinmarket.cfg`.
+
+This change is one of the reasons for the above note to recreate your `joinmarket.cfg` file. It's not appropriate to have these settings in a python source file, so the same settings are (a) now moved to a dedicated section in `joinmarket.cfg` called `[YIELDGENERATOR]` and (b) *all* of these settings can be modified on the command line when starting the maker; see the `--help` text for the script.
+
+`a490ddf` Move YG settings to a config file
+
+### Signing PSBTs
+
+With the merge of #756 it should now be possible to take a PSBT you created in another environment, where one or more utxos from a Joinmarket wallet are used as input (or a destination is taken from JM as output), and co-sign and optionally broadcast the transaction in Joinmarket. Be aware this is for now experimental, so be cautious.
```

Comment:
Here's another non-clickable PR link.

Reply:
Thanks, missed that one. There's another in a commit comment but not important.

Reply:
Done